### PR TITLE
[ru] Localize Datacenter in Russian

### PR DESCRIPTION
**Pull request by FightingFox**
### Describe your changes
Localize Datacenter in russian

### Related issue number or link (ex: `resolves #issue-number`)
https://github.com/cncf/glossary/issues/3048

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:9]: We can (and should) use the localised links whenever possible.
```suggestion
Часто дата-центры подключаются к высокоскоростным линиям связи с доступом к сети Интернет, особенно в том случае, если они ориентированы на [облачные вычисления](/ru/cloud-computing/).
```

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:30]: We actually mean the _providers_ here, not just "resources". 
```suggestion
[Публичные облака](/ru/cloud-computing/) — дата-центры, предоставляющие вычислительные ресурсы пользователям в аренду.
```

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:16]: Personally, I like the clarifications regarding the mentioned resources (e.g., «ёмкость жёсткого диска» instead of just «жёсткий диск»). However, I think we should follow the original version more strictly. Therefore, if we want these clarifications in our localised version, we'll need to propose this change in the English document first.

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:25]:
```suggestion
Из нескольких серверов создаётся [распределенная система](/distributed-systems/), функционирующая как «суперкомпьютер».
```

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:31]:
```suggestion
В последние годы существует тенденция к отказу от корпоративных дата-центров в пользу арендуемых публичных вычислительных мощностей.
```

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:15]:
```suggestion
До того момента, как в конце 1990-х годов дата-центры получили широкое распространение, компьютеры в основном использовались для решения конкретных задач или обслуживания отдельных пользователей.
```

File: content/ru/data-center.md
Comment (shurup) [inline content/ru/data-center.md:8]:
```suggestion
Дата-центр (центр хранения и обработки данных, ЦОД/ЦХОД) — здание или объект, служащий для размещения вычислительной техники, в первую очередь — серверов.
```

**Review by shurup — CHANGES_REQUESTED** [review]
Thank you for your efforts!

Here are a few corrections & suggestions from me.

File: content/ru/data-center.md
Comment (FightingFox) [reply content/ru/data-center.md:16]: Well, in the current version I will bring it in line with the English version. I will propose changes to the English-language document later.

Comment (FightingFox) [reply content/ru/data-center.md]: https://github.com/cncf/glossary/pull/3062/commits/59ad7906c83df92e28aa537235eb636f30604be3

**Comment by FightingFox** [comment]
@shurup, I complete all your changes requests. 
Check, please.

**Review by shurup — APPROVED** [review]
Thank you for all the fixes! LGTM now, but we need one more reviewer to have a look :pray: 

P.S. Before merging it, it'll also be good to squash all your commits.

File: content/ru/data-center.md
Comment (kirkonru) [inline content/ru/data-center.md:27]:
```suggestion
Суммарная производительность многих устройств позволяет запускать гораздо более масштабные приложения 
и решать более сложные вычислительные задачи.
```

File: content/ru/data-center.md
Comment (kirkonru) [inline content/ru/data-center.md:31]:
```suggestion
В последние годы наблюдается переход от корпоративных дата-центров к вычислительным мощностям в облаке.
```

**Comment by kirkonru** [comment]
Thanks for the PR!
Here are a few suggestions from me.

File: content/ru/data-center.md
Comment (FightingFox) [reply content/ru/data-center.md:27]: May be "многих устройств" -> "совокупности устройств"?
 There is an ambiguity in the version you proposed: "многих" can be perceived as "many single ones" or "most of the existing ones"

File: content/ru/data-center.md
Comment (FightingFox) [reply content/ru/data-center.md:21]: > Поскольку ресурсы отдельного компьютера (объем жёсткого диска, размер оперативной памяти, мощность процессора) лимитированы, ограничены и его возможности 
> по запуску тех или иных приложений.

My version was like this:

> Ввиду того, что каждый компьютер располагает ограниченными ресурсами (ёмкость жёсткого диска и оперативной памяти, частота и количество ядер процессора)...


But according to @shurup, this is too strong a deviation from the original source.
What will we do in the end?

File: content/ru/data-center.md
Comment (tym83) [reply content/ru/data-center.md:26]: множества устройств?

File: content/ru/data-center.md
Comment (kirkonru) [reply content/ru/data-center.md:20]: We'll wait for @tym83 🤣 
Frankly, if we're talking about resources, then those are disk _capacity_, RAM _size_, and CPU _computing power_. The disk, RAM, and CPU are the computer components.

File: content/ru/data-center.md
Comment (kirkonru) [reply content/ru/data-center.md:26]: "Совокупности" looks good to me 👍🏽

**Review by kirkonru — APPROVED** [review]

**Comment by kirkonru** [comment]
@FightingFox thanks for the great job! The pull request has been merged. 🚀

**Pull request merged**
